src/dataconnect/ensureApis.ts

```diff
@@ -7,3 +7,10 @@ export async function ensureApis(projectId: string): Promise<void> {
   await ensure(projectId, api.cloudSQLAdminOrigin(), prefix);
   await ensure(projectId, api.computeOrigin(), prefix);
 }
+
+export async function ensureSparkApis(projectId: string): Promise<void> {
+  const prefix = "dataconnect";
+  // These are the APIs that can be enabled without a billing account.
+  await ensure(projectId, api.dataconnectOrigin(), prefix);
+  await ensure(projectId, api.cloudSQLAdminOrigin(), prefix);
```

Comment:
Huh, TIL that this API doesn't require billing

Reply:
Yeah I was a little surprised by that haha

Reply:
+1